### ISSUE TITLE
Implement a learning rate finder

### ISSUE THREAD

**Issue by sebffischer** [mlr-org/mlr3torch#325]
https://arxiv.org/abs/1506.01186

I believe this might be the same as implemented here: https://mlverse.github.io/luz/reference/lr_finder.html but this needs to be checked.

Whether this is a callback or just a simple function that takes in a learner and task needs to be determined.

**Comment by sebffischer** [comment]
Also see here: https://github.com/davidtvs/pytorch-lr-finder

The interface should just be a simple function I guess.
